src/LodeRunner.UI/Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN npm run build-prod
 FROM nginx:stable-alpine AS nginx-prod
 LABEL nginx=prod
 
+RUN apk update && apk upgrade
```

Comment:
Some alternatives if we don't want to upgrade everything. At this time, both alternatives below end up installing the same version. But that could change in the future.

Upgrade only failing packages to specific version
```dockerfile
RUN apk add --no-cache \
  curl=7.83.1-r4 \
  libcurl=7.83.1-r4
```
Upgrade only failing packages to latest available version
```dockerfile
RUN apk upgrade curl libcurl
```

Reply:
Checkov did not like `RUN apk update && apk upgrade` and `RUN apk upgrade curl libcurl`.
Adding a specific version pass both Grype and Checkov scans.